src/families/tron/AccountBalanceSummaryFooter.js

```diff
@@ -150,4 +159,41 @@ const styles = StyleSheet.create({
   },
 });
 
-export default AccountBalanceSummaryFooter;
+function getInfoCandidates(t: TFunction): { [key: InfoName]: ModalInfo[] } {
+  const currency: CryptoCurrency = (findCryptoCurrencyById("tron"): any);
+  const TronIcon: React$ComponentType<{
+    color: string,
+    size: number,
+  }> = (getCryptoCurrencyIcon(currency): any);
```

Comment:
not fan of this pattern to ignore the type with a `any` cast. sounds like we are hiding flow errors and we shouldn't. I prefer // $FlowFixMe

Reply:
I cast as any and also specified const type due to Flow's limitation and its workaround.

Reply:
fixed: https://github.com/LedgerHQ/ledger-live-mobile/pull/1181/commits/a4ea5d67fa515395176b5d7c606385f95c5f105c